examples/user_guide/Pandas_API.ipynb

```diff
@@ -699,6 +706,24 @@
     "scatter_matrix(df, alpha=0.2, diagonal='kde')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since scatter matrix plots may generate a large number of points (the one above renders 120,000 points!), you man want to take advantage of the power provided by [Datashader](https://datashader.org/) to rasterize the off-diagonal plots:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.DataFrame(np.random.randn(10000, 4), columns=['a', 'b', 'c', 'd'])\n",
+    "\n",
+    "scatter_matrix(df, datashade=True, dynspread=True)"
```

Comment:
Is there a reason to use `datashade=True` here instead of `rasterize=True`? Generally we steer people towards `rasterize` unless there is a compelling reason not to, given that `rasterize` supports colorbars, hover, etc.

Reply:
No particular reason no, except that I wasn't aware of that recommendation and that I've seen instances of `datashade` being used in hvplot's docs, for instance in the Gridded data user guide.

Reply:
```suggestion
    "scatter_matrix(df, rasterize=True, dynspread=True)"
```

Reply:
There are still a couple of cases that only `datashade` covers, such as alpha-mapping an arbitrary color (with count nonlinearly mapped to the alpha channel rather than a colormap) and categorical color mixing (as in the census and ship_traffic examples). Any other case should use `rasterize`, though there are a lot of older examples written before Bokeh 2.3's datashader colormapping support that need to be updated not to use `datashade`.